### PR TITLE
build: Welcome ESM and Bye Bye UMD

### PR DESCRIPTION
**Pull request by ad1992**
Some rough notes - https://keybu.com/l/767f0bbc21a222fbda38/esm-bundle-for-excalidraw 

- [x] Ship unbundled and unmodified ESM so it can be further processed by downstream -  [try here](https://djfq22.csb.app/)
- [x] Ship bundled esm with dependencies so it can be directly used in browser via script tag - [Try here](https://tlqom.csb.app/)
- [x] Fix environment variables since `import.meta` isn't supported
- [x] Fix "index.js:735 Failed to load language en: globImport_locales_json is not defined" error
- [x] Fix webpack chunk name import errors
- [x] Remove entry.js and all unnecessary files
- [x] Remove Preact build and env vars
- [x] Remove webpack completely as its not needed anymore
- [x] Check typings should be working well
- [x] Preact should work without any additional support
 ~Make Library Tree-Shakeable~ will be coming back to this in separate PR
 ~Bundle CSS with JS for bundled ESM - not doing this since its better to load CSS via link tag as it gets loaded in parallel instead of blocking JS~

### CODE REVIEW

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

**Comment by ad1992** [comment]
@excalibot trigger release

File: scripts/buildPackage.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
+const { build } = require("esbuild");
+const { sassPlugin } = require("esbuild-sass-plugin");
+const { externalGlobalPlugin } = require("esbuild-plugin-external-global");
+
```
Comment (ad1992) [inline scripts/buildPackage.js:4]:
```suggestion
// Will be used later for treeshaking
```

**Review by dwelle — APPROVED** [review]

**Comment by ad1992** [comment]
@excalibot trigger release

File: packages/excalidraw/.gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
 node_modules
-dist
-example
 types
+bundle.js
+bundle.css
```
Comment (ad1992) [inline packages/excalidraw/.gitignore:4]:
```suggestion
bundle.css

```

**Pull request merged**